comment_extractor.go

```diff
@@ -2,15 +2,16 @@ package jsonschema
 
 import (
 	"fmt"
+	"go/ast"
+	"go/doc"
+	"go/parser"
+	"go/token"
 	"io/fs"
 	gopath "path"
 	"path/filepath"
 	"strings"
 
-	"go/ast"
-	"go/doc"
-	"go/parser"
-	"go/token"
+	"golang.org/x/exp/maps"
```

Comment:
I'm not sure that we want to be using experimental stuff...

Reply:
I can copy the implementation as it's just a straightforward `map[key]value -> []value` func.

LMK if you think this is the way.